dll/win32/shell32/iconcache.cpp

```diff
@@ -47,7 +47,11 @@ CRITICAL_SECTION_DEBUG critsect_debug =
 {
     0, 0, &SHELL32_SicCS,
     { &critsect_debug.ProcessLocksList, &critsect_debug.ProcessLocksList },
+#ifdef __REACTOS__    
+      0, 0, 0
```

Comment:
Only problem with shell32 - long long time ago in far far galaxy it was Wine code but now is not syncable. I wrapped this two lines into `__REACTOS__` ifdef